UPGRADE-4.1.md

```diff
@@ -63,6 +63,24 @@ Form
        }
    }
    ```
+ * Deprecated calling `FormRenderer::searchAndRenderBlock` for fields which were already rendered. 
+ Instead of expecting such calls return empty string, check if field has already been rendered.
+ 
+ Before:
+ ```twig
+ {% for field in fields %}
+    {{ form_widget(field) }}
+ {% endfor %}
+ ```
+ 
+ After:
+ ```twig
+ {% for field in fields %}
+   {% if not field.rendered %}
```

Comment:
This example is a bit misleading to me. Quickly reading it, you might think this check is always necessary when iterating form fields to render them.
I'd rather show an example where you need to render the same field twice and thus store the result in a variable which would be rendered twice in the "after" code.

Reply:
It's a bit misleading yes. But your suggested example is misleading too. Current code does not allow to render same field twice and neither this patch allows that.

Maybe example should just rename `fields` into `fieldsWithPossibleDuplicates`

Reply:
Alright